pkgs/applications/science/logic/coq/default.nix

```diff
@@ -5,57 +5,77 @@
 # - The exact version can be specified through the `version` argument to
 #   the derivation; it defaults to the latest stable version.
 
-{ stdenv, fetchFromGitHub, writeText, pkgconfig, gnumake42
-, ocamlPackages, ncurses
+{ stdenv, fetchzip, writeText, pkgconfig, gnumake42
+, customOcamlPackages ? null
```

Comment:
OCaml is usually spelt with a capital ‘C’.

Reply:
Will fix